.eslintrc.js

```diff
@@ -2,6 +2,7 @@ const clientESLintConfig = require('./config/eslint');
 
 module.exports = Object.assign({}, clientESLintConfig, {
   env: Object.assign({}, clientESLintConfig.env, {
-    node: true
+    node: true,
+    jest: true,
```

Comment:
This `.eslintrc.js` is only for our project (not for generated projects).
But I’m not sure it’s right—this will need to go through a few more iterations before getting merged.